### ISSUE TITLE
Open collection of baby videos demonstrating development

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#378]
**TL;DR**: Lookit could curate a collection of videos demonstrating various phenomena & concepts in child development, for family and researcher engagement.

I had this idea while taking a video of something Keoni was doing to share with a colleague because it was a cute illustration of something she studies. This would likely be separate from Lookit study video. We could provide a way on Lookit for parents to upload videos in various categories along with a short description of what's going on, and we could add some commentary. This might be a fun way for parents to engage on social media as well - we post a topic with a few examples (e.g., toddlers exerting effort, scale errors, proto-counting, variants on crawling) and people chime in with their own examples. The resulting library would be useful for teaching and outreach, potentially directly to families as well as to students in cognitive development, early education, pediatrics, etc. It could be hosted on Databrary and/or directly on Lookit.

Obviously needs scoping and I'd be interested to hear other folks' thoughts on value for recruitment/engagement as well as for teaching!

**Comment by jchu10** [comment]
I love this idea! I needed an everyday tool-use example recently and spent hours crawling through youtube / my own video data. Parents already share video on social media, and if they share it to Databrary, it would allow other parents, students, and researchers can add various tags (e.g. "persistence", "counting", "tool-use", "creative"). It does sound like something separate from Lookit, although each study could link to this sharing website and provide some example tags.

**Comment by kimberscott** [comment]
(Closing as unrelated, may still be interesting to pursue eventually)
